src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/state/actions.js

```diff
@@ -22,6 +25,26 @@ export const fetchItems = (barcode) => {
         type: SUCCESS,
         payload: response.data,
       });
+
+      if (autoCheckout && patron) {
```

Comment:
It looks like that the feature was already implemented, but maybe not working?

We should not have a `fetchItems` func to have extra responsibilities, in particular to perform the action of checking out.
Can it stay where it was? What was the issue?

Reply:
yes that is right, there was an implementation but it could can't work because the prop does not get updated in time.

Reply:
i seems to work with a small timeOut :sweat_smile: